src/components/shared/formItems/ProjectHandle.tsx

```diff
@@ -111,10 +111,10 @@ export default function ProjectHandle({
       <Input
         id="testinput"
         value={inputContents}
-        prefix="@"
+        prefix=""
         suffix={suffix}
         className="err-suffix"
-        placeholder="handle"
+        placeholder="your Juicebox username ex: @juicebox"
```

Comment:
**issue**: this component is used in a few different contexts and may not refer to the current user's project.

```suggestion
        placeholder="juicebox"
```